test_scripts/Defects/5_0/2458_Need_change_OnButtonEventNotification_logic_for_OK_button.lua

```diff
@@ -0,0 +1,80 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2458
+--
+-- Description:
+-- Need change OnButtonEventNotification logic for OK button
+-- Precondition:
+-- Core and HMI are started.
+-- In case:
+-- 1) Activate first App with HMI Level - "FULL" and second App with HMI Level "LIMITED".
+-- 2) Send On Button Event without App id, ExpectNotification only on App with "FULL" HMI level.
+-- 3) Send On Button Event with App id, ExpectNotification on App with "FULL" and "LIMITED" HMI levels. 
+-- Actual result:
+-- 1) HMI send notification only App with "FULL" HMI Level.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function hmiLeveltoLimited(pAppId)
+    common.getHMIConnection(pAppId):SendNotification("BasicCommunication.OnAppDeactivated",
+        { appID = common.getHMIAppId(pAppId) })
```

Comment:
please use the same alignment through  all script.
Check lines 25, 27 , 40, 49 and so on.